contribute/creating_package.md

```diff
@@ -96,11 +95,56 @@ To run tests on your package:
 - in your REPL, check that you are still in the repository of the project by running `pwd()`
 - type `]` which will get you into the "package mode"
 - run `activate .` to activate the environment from Project.toml
-- run `test` to perform tests
+- run `test`, or, alternatively `test Tiddlywinks` to perform tests. Test summary will provide you with the number of passed tests out of the total number of tests.
+
+## Continuous integration
+
+Continuous integration (CI) is a software development practice of merging all working copies into a mainline repository. CI tools can help you test the package on various versions of Julia, as well as various operating systems (those options are governed by the file `.travis.yml` in the case of using Travis CI).
+
+To enable CI, make sure to have pushed all of the changes to GitHub. Click on the badge in the README file in the GitHub repository of the package, and sign in with Travis CI. If you have not set up CI with Travis before, follow these steps:
+
+- on the left tab, click the `+` sign
+- on the left tab, press `Synch account`
+- on the right tab, search for the name of your newly created package. A `Settings` button should appear in front of the name of your package.
+
+Using the second badge, you can make use of Appveyor. Click on the badge, sign in with your GitHub account, authorize repositories. On the list of authorised repositories, find the name of your package, and click `Add`. You are all set to monitor changes in the package.
+
+## Propose changes via Pull Requests
```

Comment:
Here, we should link to https://kshyatt.github.io/post/firstjuliapr/ and cite Katharine

Reply:
have linked